assets/data-port/import/upload-level/index.js

```diff
@@ -0,0 +1,23 @@
+import { withSelect, withDispatch } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+import { UploadLevels } from './component';
+
+export default compose(
+	withSelect( ( select ) => {
```

Comment:
Is there a reason to use it as HOC instead of use the hooks, like used in the `data-port/import.js`? The hooks would be the modern way to do it.

Reply:
I'm liking this pattern of putting the glue code in the `index.js` with HOC, and having the pure component only responsible for the rendering. The component has a well defined list of parameters/dependencies and it's easier to read. 

Changing the file name for `component.js` to `upload-level.js` is a good idea. (All the `index.js` and similar files makes it a bit harder to navigate to things in an IDE)

Reply:
I split them up originally because it was easier to test this way, but happy to hear alternatives. Mocking `useSelect`/`useDispatch` was difficult when I couldn't do a shallow render.

Reply:
Yeah! I remembered now that at some point I also had an issue trying to mock the `useSelect` and `useDispatch` and I changed to another approach.

If you need in the future, the technique when we need to mock only some specific named exports is [using the `requireActual`](https://stackoverflow.com/a/53402206). But for some reason, it didn't work with the wordpress/data. At some point I can try to dig more to figure out what happens.

For now, it's good for me too.